recipes/bamutil/build.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 VERSION=1.0.14
-wget -O libStatGen-v$VERSION.tar.gz https://github.com/statgen/libStatGen/archive/v$VERSION.tar.gz
+curl -o libStatGen-v$VERSION.tar.gz -L https://github.com/statgen/libStatGen/archive/v$VERSION.tar.gz
 tar -xzvpf libStatGen-v$VERSION.tar.gz
+sed -i -e 's/-pg//' libStatGen-1.0.14/Makefiles/Makefile.include
```

Comment:
Can we be make this a bit more specific and descriptive, e.g.:
```bash
# remove profile flag to avoid "ld: cannot find gcrt1.o: No such file or directory"
sed -i.bak '/OPTFLAG_PROFILE?=/ s/-pg//' Makefiles/Makefile.include
```

Reply:
I added a sentence in the commit, but you are right, it is much better to keep it here.

Reply:
Ah, didn't even see that one (maybe with the excuse that GitHub collapses extended messages on default :laughing:). Yeah, having a comment beside the code is more obvious (for ppl like me).